README.md

```diff
@@ -122,6 +122,12 @@ Use this command to validate the contents of a package using the package specifi
 
 The command ensures that the package is aligned with the package spec and the README file is up-to-date with its template (if present).
 
+### `elastic-package profiles`
+
+_Context: global_
+
+
+
```

Comment:
This section is generated from code now. Please see the code for some of the other commands in the `cmd` package and then implement something similar for the `profiles` command.